abrt.spec.in

```diff
@@ -281,25 +281,31 @@ Provides: libreport-plugin-bodhi = %{version}-%{release}
 Search for a new updates in bodhi server.
 %endif
 
-%package addon-python
+%package -n python2-abrt-addon
 Summary: %{name}'s addon for catching and analyzing Python exceptions
-Requires: python
 Requires: %{name} = %{version}-%{release}
 Requires: systemd-python
-Requires: abrt-python
+Requires: python2-abrt
+# Remove before F30
```

Comment:
This is really not needed when provides/obsoletes are set correctly

Reply:
Packages should use python2- version if it is available.

https://fedoraproject.org/wiki/Packaging:Python#Dependencies